src/main/java/Duke/UserInterface.java

```diff
@@ -0,0 +1,97 @@
+package Duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class UserInterface {
+    public static void userInterface() throws DukeException {
```

Comment:
This is method is too long ([>30 LoC](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)). How about abstracting out code blocks for each case and applying [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here?